index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <title>Software Developer - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="navcontainer">
+        <nav class="navbar">
+          <div class="logo">My Logo</div>
```

Comment:
- [ ] (optional) It's important that we wrap the logo in an `a` tag as the logo is usually a link to the index page of the site for an improved user experience.

Reply:
Done.

Reply:
@Munsa1  In my ReadMe file, under Live Demo, its clearly states there that "Demo link coming soon" Do you still recommend for it to be removed?

Reply:
@Munsa1 In my index.html file, the hamburger menu is already within a nav tag. Although there is also  a div around the hamburger, I have use this for styling purposes only. Please expantiate for proper understanding.  Thanks.